internal/jimm/jimm.go

```diff
@@ -383,27 +379,36 @@ func New(p Parameters) (*JIMM, error) {
 	}
 	j.identityManager = identityManager
 
-	loginManager, err := login.NewLoginManager(j.Database, j.OpenFGAClient, j.OAuthAuthenticator, j.ResourceTag())
+	loginManager, err := login.NewLoginManager(j.Database, j.OpenFGAClient, j.OAuthAuthenticator, jimmResourceTag)
 	if err != nil {
 		return nil, err
 	}
 	j.loginManager = loginManager
 
-	permissionManager, err := permissions.NewManager(j.Database, j.OpenFGAClient, j.UUID, j.ResourceTag())
+	permissionManager, err := permissions.NewManager(j.Database, j.OpenFGAClient, j.UUID, jimmResourceTag)
 	if err != nil {
 		return nil, err
 	}
 	j.permissionManager = permissionManager
 
 	j.jujuAuthFactory = jujuauth.NewFactory(j.Database, j.JWTService, permissionManager)
 
-	auditLogManager, err := auditlog.NewAuditLogManager(j.Database, j.OpenFGAClient, j.ResourceTag(), p.AuditLogRetentionDays)
+	jujuManager, err := juju.NewJujuManager(j.Database, j.OpenFGAClient,
```

Comment:
looks like this constructor could use a parameter type.. 

also, i'd prefer something like
```
jujumanager.New(...)
```

Reply:
It could, but a few things,
- From the architecture discussion we had, the juju package might have transformers, managers, others. So probably `juju.NewManager()` is a good improvement.
- Adding a type for the constructor is fine, but not strictly necessary for this PR.
I'd like to land this and then go back in and tidy more things up.